packages/compiler/test/parser.test.ts

```diff
@@ -713,6 +714,26 @@ describe("compiler: parser", () => {
         { strict: true }
       );
     });
+
+    describe("ensure directives and decorators are applied to leaf node", () => {
+      parseEach([
+        [
+          `@doc("foo")\n#suppress "foo"\nnamespace Foo.Bar {}`,
+          (node) => {
+            const fooNs = node.statements[0];
+            assert(fooNs.kind === SyntaxKind.NamespaceStatement);
+            const barNs = (fooNs as any).statements;
+            assert(barNs.kind === SyntaxKind.NamespaceStatement);
```

Comment:
```suggestion
            strictEqual(barNs.kind, SyntaxKind.NamespaceStatement);
```
those 2 assert can change to strictEqual so it gives a better comparaison if they dont match